templates/institutional/institution-landing-en.html

```diff
@@ -0,0 +1,187 @@
+---
+title: "[Institution-landing-page]"
+layout: no-container
+language: "en"
+altLangPage: "institution-landing-fr.html"
+pageclass: "page-type-nav"
+pageType: "institution-landing"
+breadcrumb: [
+	{ "title": "Canada.ca", "link": "https://www.canada.ca/en.html" },
+	{ "title": "Departments & agencies", "link": "https://www.canada.ca/en/government/dept.html" }
+]
+dateModified: "2021-05-17"
+share: true
+noMainContainer: true
+followUsHeading: true
+followUsLayout: "inline"
+---
+<div class="provisional gc-institutional-bg mrgn-tp-md"
+	data-bgimg="https://design.canada.ca/coded-layout/images/ip-cover-image-1200x726.jpg">
```

Comment:
Can you please download the image, put it in the **/templates/institutional/img/** folder and call it from there (in both EN and FR)?